bookkeeper-server/src/test/java/org/apache/bookkeeper/client/TestGetBookieInfoTimeout.java

```diff
@@ -62,8 +65,17 @@ public TestGetBookieInfoTimeout() {
     @Before
     public void setUp() throws Exception {
         super.setUp();
-        channelFactory = new NioClientSocketChannelFactory(Executors.newCachedThreadPool(), Executors
-                .newCachedThreadPool());
+        if (SystemUtils.IS_OS_LINUX) {
```

Comment:
maybe we should create a common utility function to build EventLoopGroups and do not duplicate this kind of code